app/courses/[courseId]/student.tsx

```diff
@@ -103,9 +148,42 @@ export default function StudentCoursePage() {
         description={course.description}
         onClose={() => setInfoDialogOpen(false)}
       />
-
-      <div className="w-fit my-12 pl-8 pr-10 py-6 border-4 border-slate-200 rounded-3xl">
-        <RewardScores _scores={course.rewardScores} />
+      <div className="grid grid-cols-2">
+        <div className="w-fit my-12 pl-8 pr-10 py-6 border-4 border-slate-200 rounded-3xl">
+          <RewardScores _scores={course.rewardScores} />
+        </div>
+        <div>
+          <TableContainer component={Paper} className="my-10">
+            <Table sx={{ minWidth: 650 }} size="small">
+              <TableHead>
+                <TableRow>
+                  <TableCell sx={{ backgroundColor: "#90a4ae" }}>
+                    Student Name
+                  </TableCell>
+                  <TableCell sx={{ backgroundColor: "#90a4ae" }} align="right">
+                    Power
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {rows.map((row) => (
+                  <TableRow
+                    key={row.name}
+                    sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
+                  >
+                    <TableCell component="th" scope="row">
```

Comment:
generell fände ichs vom layout cool wenn die linke box und die rechte box irgendwie ähnlich wären und auch von der ausrichtung oben aligned sind

<img width="1501" alt="image" src="https://github.com/IT-REX-Platform/gits-frontend/assets/18666999/880d4757-e334-4ac5-98bf-b2245cb6242a">

Reply:
Die sollten jetzt beide oben gleich aligned sein. Aber mit dem Layout bin ich mir nicht sicher, was ich da machen soll. Finde es nicht schlecht wie es derzeit aussieht

Reply:
Besonders, mein Plan war es die Top drei aus dem Kurs darzustellen + man selbst, falls man nicht in den Top 3 ist. Hab es aber erst mal hinten angestellt

Reply:
so hab ich gemeint

<img width="679" alt="image" src="https://github.com/IT-REX-Platform/gits-frontend/assets/18666999/f0b3c686-c75b-4168-9702-eada7f82e6c4">

Reply:
passt denke ich kannst wegen mir mergen